pilot/docker/Dockerfile.proxyv2

```diff
@@ -14,12 +14,18 @@ COPY gcp_envoy_bootstrap.json /var/lib/istio/envoy/gcp_envoy_bootstrap_tmpl.json
 RUN chown -R istio-proxy /var/lib/istio
 
 # The following section is used as base image if BASE_DISTRIBUTION=distroless
-FROM gcr.io/distroless/cc@sha256:f81e5db8287d66b012d874a6f7fea8da5b96d9cc509aa5a9b5d095a604d4bca1 as distroless
+FROM gcr.io/distroless/cc-debian10@sha256:5bc7ef365a81ac2a58de8b9606d6840ea1aa4e03847137671671384b0b1700d4 as distroless-amd64
```

Comment:
from `:latest`

https://console.cloud.google.com/gcr/images/distroless/GLOBAL/cc-debian10?gcrImageListsize=30

Reply:
could updates when this merge https://github.com/GoogleContainerTools/distroless/pull/632
for s390x.

Reply:
`cc` => `cc-debian10` is `GLIBC_2.27` required.

https://github.com/GoogleContainerTools/distroless/issues/613